scripts/I18nPathJson-generate.js

```diff
@@ -0,0 +1,39 @@
+import { dirname, join } from "path";
+
+import * as fs from "fs";
+
+const transferLangs = ["en", "zh-CN", "ru", "ko", "pl", "uk"];
```

Comment:
```suggestion
const transferLangs = ["en", "zh-CN"];
```

I think we should shrink this list and focus on English and Chinese at the moment